test/e2e/antreapolicy_test.go

```diff
@@ -2811,22 +2811,81 @@ func TestAntreaPolicyStatus(t *testing.T) {
 		CurrentNodesRealized: 2,
 		DesiredNodesRealized: 2,
 	}
-	err = wait.Poll(100*time.Millisecond, policyRealizedTimeout, func() (bool, error) {
-		anp, err := data.crdClient.CrdV1alpha1().NetworkPolicies(anp.Namespace).Get(context.TODO(), anp.Name, metav1.GetOptions{})
+	checkANPStatus(t, data, anp, expectedStatus)
+	checkACNPStatus(t, data, acnp, expectedStatus)
+}
+
+func TestAntreaPolicyStatusWithAppliedToPerRule(t *testing.T) {
+	skipIfHasWindowsNodes(t)
+	skipIfAntreaPolicyDisabled(t)
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	server0Name, _, cleanupFunc := createAndWaitForPod(t, data, data.createNginxPodOnNode, "server-0", controlPlaneNodeName(), testNamespace, false)
```

Comment:
the function in 1.2 is defined as:
func createAndWaitForPod(t *testing.T, data *TestData, createFunc func(name string, nodeName string) error, namePrefix string, nodeName string)

I think you need to remove the last two arguments

Reply:
Fixed, thanks for reminding!

Reply:
Hi, quan. Do you think upstream should add a job to compile e2e binary, this can help to check e2e errors like this.

Reply:
I think the error was detected by several jobs? I saw many errors and e2e tests won't succeed for sure.